GenderPayGap.WebUI/Controllers/Account/ChangePasswordController.cs

```diff
@@ -61,6 +62,13 @@ public IActionResult ChangePasswordPost(ChangePasswordViewModel viewModel)
             
             // Check that passwords are valid
             ValidatePasswords(viewModel, currentUser);
+            if (currentUser.ResetAttempts == Global.MaxLoginAttempts)

```

Comment:
it might be worth setting a new global variable for Max reset attempts as well, what do you think?

Reply:
I find them somehow related (i.e. introducing wrong credentials) so that's why I used the same variable - however, if using it I should rename it.

Do you think it's worth adding a new variable for this?

Reply:
Yeah, a rename would be just as good as well, just something to show that this variable is meant to be used here, maybe something like `Global.MaxAuthAttemps`? (feel free to not use that suggestion 😅)